apps/namadillo/src/App/Transfer/__tests__/AssetImage.test.tsx

```diff
@@ -25,20 +25,20 @@ describe("Component: AssetImage", () => {
 
   it("renders the shielded badge when isShielded is true", () => {
     render(<AssetImage asset={assetMock} isShielded={true} />);
-    const badgeElement = screen.getByAltText(/Namada logo/i);
+    const badgeElement = screen.getByAltText(/Nam logo/i);
```

Comment:
I don't think this should be correct. NAM is the symbol and Namada the name of the asset

Reply:
I agree, I opened this PR to fix this on the chain registry https://github.com/anoma/namada-chain-registry/pull/13/files

Reply:
reverted :)